lib/plug/parsers.ex

```diff
@@ -200,23 +200,26 @@ defmodule Plug.Parsers do
       nil ->
         %{conn | body_params: %{}}
     end
+    merge_params(conn)
```

Comment:
If there is no parser for the content-type then `body_params` will still be unfetched when it hits this function.